vsandroid_packaging.lua

```diff
@@ -243,7 +243,7 @@
 	end
 	
 	function pack.androidAppLibName(cfg)
-		vc2010.element("AndroidAppLibName", nil, "$(RootNamespace)")	-- TODO: fix
+		vc2010.element("AndroidAppLibName", nil, cfg.androidapplibname)
```

Comment:
It's tempting to have this configurable per config to enable the range of configuration options that VS allows. On the other hand, `manifest` is currently a project-wide setting, so I'll mirror this with `applibname` until I have enough spare time on my hand to evaluate updating `manifest` (in a separate PR).